ios/application_swiftui/ContentView.swift

```diff
@@ -1,8 +1,45 @@
 import SwiftUI
 
 struct ContentView: View {
-	var body: some View {
-		Text("Hello, world!")
-			.padding()
-	}
+    @State private var items = [Date]()
```

Comment:
nit: The `iphone/application_swift` template uses the following syntax:

https://github.com/theos/templates/blob/78f54b96aa0018bb7bd30000965cc07b403ab576/ios/application_swift/RootViewController.swift#L5

Do we have a preference?

Reply:
Let's use matching syntax for consistency

```suggestion
    @State private var items: [Date] = []
```